src/object_finding/feature_detector.cpp

```diff
@@ -0,0 +1,46 @@
+#include "feature_detector.hpp"
+
+namespace lucidy{
+
+    FeatureDetector::FeatureDetector(settings::FDT::data & settings):
+        settings(settings)
+    {initDetector();}
+
+    void FeatureDetector::initDetector(){
```

Comment:
Place the content of initDetector in constructor

Reply:
This makes code unnecessarily long and init function is alway more recommend. Invalid comment due to bad advice